cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesModel.java

```diff
@@ -22,6 +23,8 @@
  */
 public interface CgmesModel {
 
+    TripleStore tripleStore();
```

Comment:
getTripleStore?

Reply:
This methods was already present in implementation class and I didn't want to change it.
I added a missing @Override in impl class.